.github/CODEOWNERS

```diff
@@ -114,11 +108,12 @@
 # ServiceLabel: %Communication - Common
 /sdk/communication/communication-common/ @AikoBB @maximrytych-ms @mjafferi-msft
 
-# AzureSdkOwners: @timovv @jeremymeng
 # PRLabel: %Container Registry
-# ServiceLabel: %Container Registry
 /sdk/containerregistry/ @timovv @jeremymeng @Azure/azsdk-acr
 
+# ServiceLabel: %Container Registry
+# AzureSdkOwners: @timovv @jeremymeng
```

Comment:
No service team members?   We'll end up with a gap if we ever use "Service Attention" with this.

Reply:
It's been quite in ACR space.  We will figure it out when there's a need.  So as far as the automation is concerned, would that mean we don't have auto-triage for ACR, or anything else?

Reply:
Autotriage will assume its owned by us.   Its just "Service Attention" that will fail.

Reply:
I see, if we don't use "Service Attention" for ACR we would be fine.